default/methods/account/plan/usage_cycle.py

```diff
@@ -260,7 +260,7 @@ def determine_billable_directory_file_and_instance(
 
```

Comment:
See https://github.com/diffgram/diffgram/pull/1536 for comments explaining diffs